en/docs/integration-guides/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```diff
@@ -0,0 +1,601 @@
+      Automation integrations can be triggered **manually** or **periodically**, and are ideal for scheduled file transformations.
+
+
+  3. Create the Automation
+
+      A page titled **Automation – An automation that can be invoked periodically or manually** appears.  
+      Click the blue **Create** button to add a new Automation flow.  
+      You may optionally expand *Optional Configurations* to adjust scheduling or trigger settings, but for this tutorial we keep defaults.
+
+      The tool shows a brief **“Creating…”** indicator, and then a new canvas opens.
+
+
+  4. Observe the Automation Canvas
+
+      Once created, the visual editor displays a blank automation diagram with a starting node.  
+      This represents the root of your pipeline where you will define the steps to read, map, and write data.
+
+
+## Step 6: Add a file read function to the automation flow
+
+In this step, you’ll configure the automation’s first functional node — a **Call Function** block — to read CSV data from the configured input path (`inputCSV`) into an array of CSV records (`CSV[]`).  
+This forms the initial data acquisition step of the workflow.
+
+
+  1. Open the Automation Flow
+
+      With the automation already created, you’ll now see a **Start** node connected to an **Error Handler** node.  
+      This is the default template for new automations.
+
+
+  2. Add the First Node
+
+    1. Hover your mouse below the **Start** node until a small **“+” button** appears.  
+    2. Click **+** to add a new operation.  
+        The system briefly displays `Generating next suggestion...` — this means it’s fetching available node types.
+
+
+  3. Choose the Node Type
+
+      Once suggestions appear, look to the **right-hand Node Panel** under the **Statement** category.  
+      You’ll see options such as:
+
+      - Declare Variable  
+      - Assign  
+      - **Call Function**  
+      - Map Data  
+
+Click **Call Function** to insert this node into the automation flow.
+
+
+## Step 7: Use `fileReadCsv` to load the csv file into memory
+
+Now that your Call Function flow is open, the next step is to configure a CSV File Read Function using the built-in `io:fileReadCsv()` operation.  
+This function reads CSV data from the path defined in your project’s configurables and loads it as a structured array of `CSV` records that can be mapped later.
+
+
+  1. Open the Function Search Panel
+
+    In the right-side **Functions** panel, scroll or search for a CSV-related function:
+    1. Click inside the **Search library functions** field.  
+    2. Type **csv** to filter the standard library.  
+    3. You’ll see a list under **data.csv** and **io** modules containing functions like:
+      - `parseStream`
+      - `parseString`
+      - `fileReadCsv`
+      - `fileWriteCsv`
+
+
+  2. Select `io:fileReadCsv`
+
+    Click **fileReadCsv** from the results list.  
+    A configuration panel opens on the right describing the function:
```

Comment:
Avoid using unessary new lines. This line can be compned with previous line. Check all the places.